### ISSUE TITLE
Any plans for L7 path routing?

### ISSUE THREAD

**Issue by scalp42** [yyyar/gobetween#284]
Hi folks,

This looks very interesting. We're currently using [Fabio](https://github.com/fabiolb/fabio#notes) and we're exploring other options.

It doesn't seem that gobetween handles path routing (Fabio looks for tags `fabio-/mypath` for example) or path stripping.

Is it in the plans or will it be a L4 solution only?

Thanks for gobetween!

**Comment by nickdoikov** [comment]
Yes, there are some plans regarding L7. But it should be properly designed that's why it is not still here. :(
